src/pages/how-it-works.js

```diff
@@ -0,0 +1,48 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import Hero from "@/pesayetu/components/OtherHero";
+import Page from "@/pesayetu/components/Page";
+import formatBlocksForSections from "@/pesayetu/functions/formatBlocksForSections";
+import getPostTypeStaticProps from "@/pesayetu/functions/postTypes/getPostTypeStaticProps";
+
+export default function Home({ blocks, ...props }) {
+  return (
+    <Page {...props}>
+      <Hero {...blocks?.otherHero} />
+    </Page>
+  );
+}
+
+Home.propTypes = {
+  blocks: PropTypes.shape({
+    otherHero: PropTypes.shape({}),
+  }),
+};
+
+Home.defaultProps = {
+  blocks: undefined,
+};
+
+export async function getStaticProps() {
```

Comment:
```suggestion
export async function getStaticProps({ preview, previewData }) {
```
Can we please add this to homepage too